migrations/upgrade-1-8-0.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Migration to create Smaily widget settings.
+ */
+
+$upgrade = function() {
+
+	$widgets = get_option( 'widget_smaily_widget', array() ); // Array sets empty default value.
+	foreach ( $widgets as &$widget ) {
+		if ( ! is_array( $widget ) ) {
+			continue;
+		}
+		$array2 = array(
+			'form_layout'                       => 'layout-5',
+			'email_field_placeholder'           => __( 'Email', 'smaily' ),
+			'name_field_placeholder'            => __( 'Name', 'smaily' ),
+			'submit_button_text '               => __( 'Send', 'smaily' ),
+			'submit_button_color'               => '',
+			'submit_button_text_color'          => '',
+			'use_site_submit_button_color'      => true,
+			'use_site_submit_button_text_color' => true,
+		);
```

Comment:
You are not applying new settings to existing widgets.

```suggestion
		);
		$widget = array_merge( $widget, $array2 );
```